global.json

```diff
@@ -8,7 +8,7 @@
     }
   },
   "native-tools": {
-    "cmake": "3.21.0"
+    "cmake": "latest"
```

Comment:
AFAIK native-tools aren't supported anymore by Arcade. But I might be wrong.

Reply:
They're on the machine natively, actived with https://github.com/dotnet/installer/pull/18525/files#diff-8376b15ac9abd63c996f1b3abd9386026b091e774b7e55ccddcf10485077b83fR35